EcpBtBot/DialogFlow/Response.py

```diff
@@ -47,4 +47,3 @@ class Status:
     def __init__(self,status):
```

Comment:
mutliple `__init__` is not recommanded in python. better add status and result as an optional parameter: 

```
 @def __init__(self, result=None, status=None):
  if result is not None:
    self.resolvedQuery = result.get('resolvedQuery', None)
    ...
  if status is not None:
    self.code = status.get('code', None)
    self.error = status.get('errorType', None)
```